clients/java/signalr/src/main/java/IHubProtocol.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public interface IHubProtocol {
+    String name();
+    int version();
```

Comment:
Is this the naming convention for this kind of thing? I vaguely remember using the `getVersion()` style way back when I last touched Java but maybe that's changed.

Reply:
Should be `getVersion`